linux/software/libcamera/README.md

```diff
@@ -9,71 +9,99 @@ sudo apt update
 sudo apt full-upgrade
 ```
 
-Currently (May 2020) the necessary _libcamera_ support has not yet been merged into the standard Raspberry Pi OS release, therefore it is necessary to install the latest release candidate. To do this, first reboot your Pi, and then use
+libcamera is under active development which sometimes means that new features need to be supported in Raspberry Pi OS, even before they are officially released. Therefore we currently recommend updating to the latest release candidate. To do this, first reboot your Pi, and then use
 
 ```bash
 sudo rpi-update
 ```
 
 **WARNING**: Note that the release candidate is not as thoroughly tested as an official release. If your Raspberry Pi contains important or critical data we would strongly advise that it is backed up first, or that a fresh SD card is used for the purpose of trying _libcamera_.
 
-Next, the `/boot/config.txt` file must be updated to load and use the camera driver, by adding the following lines to the bottom. Currently we also need to update the GPU's `core_freq_min` though this will become unnecessary in due course after further updates.
+Next, the `/boot/config.txt` file must be updated to load and use the camera driver, by adding the following to the bottom.
 
 ```bash
 dtoverlay=imx219
-core_freq_min=250
 ```
 
-If you are using a sensor other than the `imx219` you will need to supply the alternative name here (for example, `ov5647` for the V1 camera, or `imx477` for the HQ Cam for which support will be available shortly).
+If you are using a sensor other than the `imx219` you will need to supply the alternative name here (for example, `ov5647` for the V1 camera, or `imx477` for the HQ Cam).
 
 **NOTE**: after rebooting, control of the camera system will be passed to the ARM cores, and firmware-based camera functions (such as raspistill and so forth) will no longer work. Setting `/boot/config.txt` back and rebooting will restore the previous behaviour.
 
-## Software Dependencies
+## Building _libcamera_ and _qcam_
 
 The build system and runtime environment of _libcamera_ have a number of dependencies. They can be installed with the following commands.
 
 ```bash
 sudo apt install libboost-dev
 sudo apt install libgnutls28-dev openssl libtiff5-dev
-sudo apt install meson
 sudo apt install qtbase5-dev libqt5core5a libqt5gui5 libqt5widgets5
-sudo pip3 install pyyaml
+sudo apt install meson
+sudo pip3 install pyyaml ply
 ```
 
-## Building _libcamera_ and _qcam_
+The Qt libraries are only required for _libcamera_'s _qcam_ demo app.
 
-We can now check out the code and configure the build for the Raspberry Pi as follows.
+Unfortunately, at the time of writing, the default version of meson is a little old, so please execute:
 
 ```bash
-git clone git://linuxtv.org/libcamera.git
-cd libcamera
-```
+wget http://ftp.br.debian.org/debian/pool/main/m/meson/meson_0.56.2-1_all.deb
+sudo dpkg -i meson_0.56.2-1_all.deb
+ ```
 
```

Comment:
Not sure it matters, but I did a
`sudo pip3 install --update meson`

to get the latest version.

Reply:
Oooh, that's much nicer, didn't know you could do that. I'll push that correction.

Reply:
Though the magic seems to require "--upgrade", not "--update" !!

Reply:
Oops, yes.  Wrote that (erroneously) from memory.

Reply:
Age. Once you get to my age, you won't even remember your children's names! Anyway, must go, child 2 needs something...